schemas/gathering_rules.schema.json

```
@@ -0,0 +1,38 @@
+{
+  "$id": "gathering_rules.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "array of rules for conditional gatherer",
+  "descriptions": "Gathering rules for conditional gatherer",
```

I just realized that `description` vs. `descriptions` is probably older than I thought. :grinning: 

Could you fix it here as well, please?

```suggestion
  "description": "Gathering rules for conditional gatherer",
```